### ISSUE TITLE
Suggestion: State that GPU is not used for inference

### ISSUE THREAD

**Issue by ajarang** [PromtEngineer/localGPT#668]
I spent a lot of time wondering why my GPU (4080 laptop 12GB) was not being used when running LocalGPT, even when I could see that the localGPT was running on cuda. However, thanks to this [issue](https://github.com/PromtEngineer/localGPT/issues/597), it seems that answer generation doesn't happen on GPU, just that the weights are loaded in cuda. Is this true? If so, it would be good to specify in the readme.

**Comment by ajarang** [comment]
Turns out that I'm an idiot and the issue that I linked was misleading. On windows, I had to reinstall llama-cpp-python using all the cmake arguments (for windows 10 console, within anaconda)
`set CMAKE_ARGS=-DLLAMA_CUBLAS=on
set FORCE_CMAKE=1
pip install llama-cpp-python
`
I kept getting a install wheel issue, which typically means that the visual studio build tools was not installed properly, even though I verified that all the files I needed were installed. Turns out I also needed to drag everything from my cuda extensions folder to my visual studio build customizations folder: "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.1\extras\visual_studio_integration\MSBuildExtensions" to "C:\Program Files (x86)\Microsoft Visual Studio\2022\BuildTools\MSBuild\Microsoft\VC\v170\BuildCustomizations". 

Afterward everything worked well- hope that this helps someone out
